spec/support/features/admin_user_helpers.rb

```diff
@@ -0,0 +1,19 @@
+module Features
+  module AdminUserHelpers
+    def sign_in_to_admin_panel(admin_user)
+      visit '/admin'
+      fill_in 'admin_user_email', with: admin_user[:email]
+      fill_in 'admin_user_password', with: admin_user[:password]
+      click_button 'Log in'
+    end
+
+    def fill_event_form(event)
```

Comment:
We must not couple this method to the event object, let's just pass parameters
```ruby
def fill_event_form(event_params = {})
```

Reply:
@alexshgov I'm not sure if that's what you wanted, but I got rid of the object